src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.module.scss

```
@@ -5,29 +5,57 @@
   flex-direction: column;
   gap: $spacing-md;
 
-  h3 {
-    font: $text-title-3xs;
+  .sectionTitle {
+    h3 {
+      font: $text-title-3xs;
+    }
+
+    p {
+      color: $color-grey-40;
+    }
   }
 
   .radioGroup {
     display: flex;
     flex-direction: column;
-    gap: $spacing-sm;
+    gap: $spacing-md;
 
-    label {
+    .emailAlertsOptions {
+      padding-left: $spacing-lg;
```

Nit, because I don't think we support them, but good to know: to have the spacing automatically adjust in right-to-left-languages, you can use [CSS logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values):

```suggestion
      padding-inline-start: $spacing-lg;
```